CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changelog - chai-webdriverio
+
+## Unreleased Changes
+#### Breaking Changes:
+- Changed `there` and `visible` from methods to properties:
```

Comment:
Oh, I suppose `to.be` would be defined, and visible would just return `undefined` huh?  Dang.  Another reason they have the `dom` flag I guess!  Man, having that flag seems more and more useful...